lib/adal.js

```diff
@@ -1187,7 +1141,7 @@ AuthenticationContext.prototype._cloneConfig = function (obj) {
     return copy;
 };
 
-AuthenticationContext.prototype._addClientId = function () {
+AuthenticationContext.prototype._addXClientParameters = function () {
```

Comment:
Since both of these values represent information about the library, sku==platform, ver==version, another names could be _addLibraryMetadata or _addMetadata.